gourmet/backends/db.py

```diff
@@ -1,3 +1,5 @@
+from gi.repository import Gtk
+from functools import cmp_to_key
```

Comment:
Unused import (I think I removed its use in another PR recently)

Reply:
I missed that `cmp_to_key` was removed when I resolved the conflict.